commons/src/test/java/org/eclipse/kapua/commons/util/KapuaFileUtilsTest.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.commons.util;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.net.URL;
+
+@Category(JUnitTests.class)
+public class KapuaFileUtilsTest extends Assert {
+
+    @Test
+    public void constructorTest() throws Exception {
+        Constructor<KapuaFileUtils> fileUtilsConstructor = KapuaFileUtils.class.getDeclaredConstructor();
+        fileUtilsConstructor.setAccessible(true);
+        fileUtilsConstructor.newInstance();
+    }
+
+    @Test
+    public void getAsUrlPositiveAndNegativeTest() throws Exception {
+        String[] permittedFormats = new String[] {"https://www.example3345.com/", "http://www.example.com:1080/docs/resource1.html",
+                "https://www.example.com:1080/docs/resource2.html", "file:///c:/EXAMPLE/clock.example"};
+        String [] invalidFormats = new String [] {"www.example.com:1080/docs/resource1.html", "/c:/WINDOWS/clock.example",
+                "http://www.example.com:10800000000000/docs/resource1.html", "htp://www.example.com:1080/docs/resource1.html"};
+
+        URL[] expectedUrl = new URL[] {new URL("https://www.example3345.com/"), new URL("http://www.example.com:1080/docs/resource1.html"),
+                                       new URL("https://www.example.com:1080/docs/resource2.html"), new URL("file:///c:/EXAMPLE/clock.example")};
+        for (int i = 0; i < permittedFormats.length; i++) {
+            try {
+                assertEquals(expectedUrl[i], KapuaFileUtils.getAsURL(permittedFormats[i]));
+            } catch (Exception ex) {
+                fail("Inappropriate http scheme.");
+            }
+            try {
+                assertEquals(invalidFormats[i], KapuaFileUtils.getAsURL(invalidFormats[i]));
```

Comment:
If they are not intended to match please use `assertNotEquals`. 
If `KapuaFileUtils.getAsURL(...)` is meant to throw an Exception catch the Exception and do asserts on the type of Exception expected.